doc/api/process.md

```diff
@@ -422,6 +422,9 @@ It is important to take note of the following:
 * `SIGKILL` cannot have a listener installed, it will unconditionally terminate
   Node.js on all platforms.
 * `SIGSTOP` cannot have a listener installed.
+* `SIGBUS`, `SIGFPE`, `SIGSEGV` and `SIGILL`, when not raised artificially,
+   inherently leave the process in a state from which it is not safe to
+   attempt to call JS listeners.
```

Comment:
> and that works perfectly when the signal is artificially generated,

Does that imply that we should not allow these signals to trigger when they are not artificial?